src/lib.rs

```diff
@@ -138,6 +138,21 @@ struct Registers {
     _reserved13: [u8; 3],
 }
 
+/// PL011 Errors
+#[derive(Debug, PartialEq)]
+pub enum Error {
+    FramingError,
+    ParityError,
+    BreakError,
+    OverrunError,
+}
+
+impl embedded_io::Error for Error {
+    fn kind(&self) -> embedded_io::ErrorKind {
+        embedded_io::ErrorKind::Other
```

Comment:
Import `ErrorKind` so it doesn't need to be fully-qualified here.

Reply:
Done